mysql-test/suite/rpl/t/rpl_xa_prepare_gtid_fail.test

```diff
@@ -25,9 +25,9 @@ source include/have_debug_sync.inc;
 --connection slave
```

Comment:
While we're here, let's shift l.19 `master-slave.inc` to the end of the sourced inc:s.